library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp

```diff
@@ -1582,8 +1785,31 @@ rocblaslt_status
     rocblaslt_status status = rocblaslt_status_success;
     try
     {
-        status = getBestSolutions(
-            handle, gemmType, gemmData, workspaceBytes, requestedAlgoCount, results);
+        OverrideSingleton& override = OverrideSingleton::getInstance();
+        bool override_success = false;
+        std::vector<rocblaslt_matmul_heuristic_result> override_result;
+
+        if (override.env_mode)
+        {   
+            override_success = problem_override_from_file_cpp(handle,
+                                                              gemmType,
+                                                              gemmData,
+                                                              workspaceBytes,
+                                                              override_result, 
+                                                              override.file_path);
+                                                              
+            log_api(__func__, "returnAlogCount", override_success ? 1 : 0);
+
+        }
+        
+        if (requestedAlgoCount - override_result.size() > 0)
+            status = getBestSolutions(handle, gemmType, gemmData, workspaceBytes, override_success ? requestedAlgoCount - 1 : requestedAlgoCount, results);
```

Comment:
have your run clang-format?

Reply:
it’s done in commit [67ab4a4](https://github.com/ROCm/hipBLASLt/pull/1289/commits/67ab4a4d7081fc718de442762c13e36491ce2c47)